src/Relude/Extra/Bifunctor.hs

```diff
@@ -50,3 +51,19 @@ Just (False,2)
 secondF  :: (Functor f, Bifunctor p) => (b -> d) -> f (p a b) -> f (p a d)
 secondF = fmap . second
 {-# INLINE secondF #-}
+
+{- | Maps a function over both elements of a bifunctor.
+
+>>> bimapBoth length ("a", "bb")
+(1,2)
+>>> bimapBoth length ("foo", "a", "bb")
+("foo",1,2)
+>>> map (bimapBoth not) [Left True, Right False]
+[Left False,Right True]
+>>> import Data.Functor.Const
+>>> getConst $ bimapBoth (+ 1) (Const 41)
+42
```

Comment:
Let's remove examples for triple and `Const` to decrease confusion :slightly_smiling_face: In most cases this function is going to be used with a pair or `Either`. So let's keep focus of documentation readers only on relevant examples. Experienced Haskell developers, who are familiar with `Bifunctor`, can generalise this function in their head.